service-front/module/Application/src/Form/Validator/Phone.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Application\Form\Validator;
+
+use Laminas\Validator\AbstractValidator;
+
+class Phone extends AbstractValidator
+{
+    private const string NOT_PHONE = 'notPhone';
+
+    /**
+     * @var array<string, string>
+     */
+    protected array $messageTemplates = [
+        self::NOT_PHONE => 'Enter a valid phone number',
+    ];
+
+    public function isValid($value)
+    {
+        $normalizedValue = preg_replace('/\s+/', '', (string) $value);
+        $this->setValue($normalizedValue);
+
+        $pattern = '/^(\+)?(\d|\(|\)| |\-){6,20}$/';
```

Comment:
I've favoured readability over being 100% correct. This would allow users to enter between 6 and 20 of either `(`, `)` or `-` but I'm assuming we don't really care if that happens?